### PR TITLE
Auto fill email and verification token fields on register/signup routes

### LINKED ISSUE #1082: [cms] Add a new route for invitiatiion links
Links in invitation emails are as follows:

`/register?email=ay-p@example.org&token=<token>`

This should resolve to the RegisterForm page and have the email and token fields auto populated.  We could do something fancier in the future and go to a new form that only takes username, password and verify password.  But for MVP, just the auto-population is fine.

### PR DESCRIPTION
**Pull request by tiagoalvesdulce**
Closes #1082

### CODE REVIEW

**Comment by alexlyp** [comment]
Hrm the token isn't populating for me, only the email address currently

**Comment by tiagoalvesdulce** [comment]
Can you share the URL you are using to test? Here works fine with `/register?email=ay-p@example.org&token=<token>`. The email field is filled with `ay-p@example.org` and the invitation token is filled with `<token>`.

**Comment by alexlyp** [comment]
https://localhost:3000/register?email=email%40email.com&verificationtoken=279edfcf5d422284d24690d9e8913e5be7f851631a4b87c4f7a1409ba169555a

**Comment by alexlyp** [comment]
Whoops yah I mistaken said it was just "token" but it's actually verificationtoken in the link.

Worked when I changed it to just token

File: src/hocs/signupForm.js [hunk 1]
```diff
@@ -97,8 +97,15 @@ const wrap = Component =>
     return <Comp />;
   });
 
+const { email, token: verificationtoken } = getQueryStringValues();
```
Comment (alexlyp) [inline src/hocs/signupForm.js]: can just remove the "token:" I believe, sorry about that!

**Pull request merged**